projects/ngx-sub-form/src/lib/ngx-sub-form-utils.ts

```diff
@@ -1,28 +1,35 @@
 import {
-  AbstractControl,
   ControlValueAccessor,
   NG_VALUE_ACCESSOR,
   NG_VALIDATORS,
   ValidationErrors,
+  FormControl,
+  FormArray,
 } from '@angular/forms';
 import { InjectionToken, Type, forwardRef, OnDestroy } from '@angular/core';
 import { Observable, Subject, timer } from 'rxjs';
 import { takeUntil, debounce } from 'rxjs/operators';
 import { SUB_FORM_COMPONENT_TOKEN } from './ngx-sub-form-tokens';
 import { NgxSubFormComponent } from './ngx-sub-form.component';
 
-export type Controls<T> = { [K in keyof T]-?: AbstractControl };
+export type Controls<T> = { [K in keyof T]-?: FormControl | FormArray };
```

Comment:
Is it true that we don't support form group? This would be a breaking change...

Reply:
> Is it true that we don't support form group?

We technically do... But I think that it should never happen has ngx-sub-form is here to help break things down so the only valid `FormGroup` IMO is the one we handle internally.

> This would be a breaking change...

Yes, maybe for later then!